pkg/app/app_deploy.go

```diff
@@ -56,6 +57,15 @@ func installAppGroup(captenConfig config.CaptenConfig, hc *helm.Client, appConfi
 		} else {
 			clog.Logger.Infof("[app: %s] installed", appConfig.Name)
 		}
+		if alreadyInstalled && (captenConfig.ClusterType == "cloud-managed") && (appConfig.Name == "traefik") {
+			hostName, err := k8s.PrintLoadBalancerServices(captenConfig.PrepareFilePath(captenConfig.ConfigDirPath, captenConfig.KubeConfigFileName), "traefik")
+			if err != nil {
+				clog.Logger.Error("failed to get LoadBalancerService ", err)
+			}
+			config.UpdateLBEndpointFile(&captenConfig, hostName)
+			time.Sleep(3 * time.Minute)
+
+		}
```

Comment:
move this separate action, it can't wait on traefik app deploying.

like store secrets action, add action getch loabbalancer config and update to file.
use retry function block that check every 30 sec.